src/shared/components/feature-summary-strip/VariantSummaryStrip.tsx

```diff
@@ -70,28 +77,36 @@ const VariantSummaryStrip = (
 };
 
 const FullContent = ({ variant }: { variant: VariantForSummaryStrip }) => {
+  const { referenceAllele } = getReferenceAndAltAlleles(variant.alleles);
   const mostSevereConsequence = (
     <VariantConsequence variant={variant} withColour={false} />
   );
 
   return (
     <>
-      <div className={styles.section}>
-        <span className={styles.featureSummaryStripLabel}>Variant</span>
-        <span className={styles.featureNameEmphasized}>{variant.name}</span>
+      <div className={featureStripStyles.section}>
+        <span className={featureStripStyles.featureSummaryStripLabel}>
+          Variant
+        </span>
+        <span className={featureStripStyles.featureNameEmphasized}>
+          {variant.name}
+        </span>
+        <span className={styles.variantType}>
+          {referenceAllele?.allele_type.value}
```

Comment:
If you do this, then the label will always read "biological_region" (this is what the type of the reference allele always is).

What you should use instead is `variant->allele_type->value`